package.json

```diff
@@ -6,36 +6,41 @@
     "type": "git",
     "url": "https://github.com/NYPL/nypl-design-system/"
   },
-  "main": "dist/design-system-react-components.min.js",
-  "types": "lib/index.d.ts",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "module": "dist/design-system-react-components.esm.js",
+  "files": [
+    "dist",
+    "src"
+  ],
+  "engines": {
+    "node": ">10"
+  },
   "author": "NYPL",
   "license": "Apache-2.0",
   "publishConfig": {
     "access": "public"
   },
   "scripts": {
-    "start": "node node_modules/@storybook/react/bin/index -p 9001 -c .storybook",
-    "build-storybook": "node node_modules/.bin/build-storybook -c .storybook -o ./storybook-static",
-    "test": "npm run lint && npm run test:ci",
-    "test:watch": "jest --watch",
-    "test:ci": "jest",
-    "format": "prettier --write 'src/**/*.{ts, tsx, js,jsx,json}'",
-    "pretty-quick": "pretty-quick",
-    "prepare": "npm run dist",
+    "start": "tsdx watch",
+    "build": "tsdx build && npm run generate-sass-resources",
+    "test": "tsdx test --passWithNoTests",
+    "test:watch": "tsdx test - --watch",
+    "lint": "tsdx lint",
+    "prepare": "npm run build",
+    "size": "size-limit",
+    "analyze": "size-limit --why",
     "generate-sass-resources": "gulp",
```

Comment:
Was there also a desire to remove gulp?  Is that happening in a different chunk of work?

Reply:
Yes, I haven't found a way to do what gulp does in webpack or rollup that does _two_ processes for the scss files. What I've done so far is try a roll plugin that runs in the same pipeline as the `scss` plugin. It does correctly read an input file and combines all the relevant code into a final `.scss` file, but it can't do that _and_ create the minified `dist/styles.css` file. There is also a separate ticket for this (existing before this conversion) that will be relevant to this.

Reply:
Yeah - also I wonder if this is going to change after the chakra migration, it's not clear how much of our custom SCSS we're going to keep.